tests/01_eradiate/03_regression/atmospheres/test_rpv_afgl1986_continental.py

```diff
@@ -61,7 +61,26 @@ def test_rpv_afgl1986_continental_brfpp(
         },
         atmosphere={
             "type": "heterogeneous",
-            "molecular_atmosphere": {"type": "molecular", "construct": "afgl_1986"},
+            "molecular_atmosphere": {
+                "type": "molecular",
+                "thermoprops": {
+                    "identifier": "afgl_1986-us_standard",
+                    "z": np.arange(0, 120.1, 0.1) * ureg.km,
+                    "additional_molecules": False,
+                },
+                "absorption_data": (
+                    "spectra/absorption/ckd/monotropa/monotropa-18100_18200.nc"
+                ),
+                "error_handler_config": {
+                    "x": {
+                        "missing": "ignore",
+                        "scalar": "ignore",
+                        "bounds": "raise",
+                    },
+                    "p": {"bounds": "ignore"},
+                    "t": {"bounds": "ignore"},
+                },
```

Comment:
Because this is a regression test and I thought setting explicitly this config would be safer than using some default defined elsewhere, since in regression test we are concerned with the problem that the same input should give the same output.